tests/framework/installer/ceph_manifests_v1.4.go

```diff
@@ -1806,24 +2022,68 @@ parameters:
 `
 }
 
-func (m *CephManifestsV1_2) GetPVC(claimName, namespace, storageClassName, accessModes, size string) string {
+func (m *CephManifestsV1_4) GetPVC(claimName, namespace, storageClassName, accessModes, size string) string {
 	return `apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
   name: ` + claimName + `
   namespace: ` + namespace + `
-  annotations:
-    volume.beta.kubernetes.io/storage-class: ` + storageClassName + `
 spec:
+  storageClassName: ` + storageClassName + `
   accessModes:
     - ` + accessModes + `
   resources:
     requests:
       storage: ` + size
 }
 
+func (m *CephManifestsV1_4) GetBlockSnapshotClass(snapshotClassName, namespace, systemNamespace, reclaimPolicy string) string {
+	return ``
```

Comment:
nit: no need for the multi-line string markers
```suggestion
	return ""
```